### ISSUE TITLE
Build failing due to error in ngx-datepicker

### ISSUE THREAD

**Issue by thinker3197** [kekeh/ngx-mydatepicker#1]
Thanks for this great module. However, the build is failing because of the module. 

**Angular cli**
```
> ng -v 

@angular/cli: 1.0.0-beta.30
node: 6.9.1
os: darwin x64
@angular/common: 2.4.6
@angular/compiler: 2.4.6
@angular/core: 2.4.6
@angular/forms: 2.4.6
@angular/http: 2.4.6
@angular/platform-browser: 2.4.6
@angular/platform-browser-dynamic: 2.4.6
@angular/router: 3.4.6
@angular/cli: 1.0.0-beta.30
@angular/compiler-cli: 2.4.6
```

**Error during build**
```
ERROR in NgxMyDatePickerModule is not an NgModule
webpack: Failed to compile.
```

Any help would be appreciated! Thanks

**Comment by kekeh** [comment]
Hello @thinker3197 

I tried ngx-mydatepicker with **@angular/cli: 1.0.0-beta.26** and **@angular/cli: 1.0.0-beta.32.3**. With date picker the build and the AOT build worked ok in both versions.

I don't know why it is failing in your project. How you import the NgxMyDatePickerModule module in your NgModule?

Did you followed instructions from here: https://github.com/kekeh/ngx-mydatepicker#installation

**Comment by thinker3197** [comment]
Yes, I followed the same instructions. Moreover, I tried upgrading `@angular/cli` to `1.0.0-beta.32.3` and it gives the same error on serve and fails to build. I don't think it's something specific to this library, looks like an angular-cli [issue](https://github.com/angular/angular-cli/issues/3426).

**Comment by kekeh** [comment]
I tested version **@angular/cli: 1.0.0-beta.30** and it worked also ok.

Did you check this? https://github.com/angular/angular-cli/issues/4594

**Comment by AlwaysAbhl001** [comment]
@thinker3197  how could you solve it? facing same issue.

**Comment by densanki** [comment]
Also face problem when npm dependency is not loaded.

Version "ngx-mydatepicker": "^2.0.12"
Angular CLI Version 1.1.2
NPM Version 5.0.4

We had the problem that register of module was successful on the node the install was made. But on other nodes the module is missing after install.

ERROR in src/app/admin/shared/conversion.service.ts (2,23): Cannot find module 'ngx-mydatepicker'.
